src/core/InterleavedBufferAttribute.js

```diff
@@ -5,27 +5,18 @@ const _vector = new /*@__PURE__*/ Vector3();
 
 class InterleavedBufferAttribute {
 
-	constructor( interleavedBuffer, itemSize, offset, normalized ) {
+	constructor( buffer, itemSize, type, normalized, stride, offset, count ) {
 
 		this.name = '';
 
-		this.data = interleavedBuffer;
+		this.data = buffer;
+		this.array = new type( buffer.array.buffer, offset );
```

Comment:
Note that in addition to the requirement that stride be a multiple of component size, this also requires that offset be a multiple of 4 I believe. May need to document these alignment requirements.

Reply:
Sounds good to me unless other specs need.

Reply:
Requirement of Float32Array constructor itself as well: https://stackoverflow.com/a/8638875/1314762

Reply:
Thanks, I didn't know that.